<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DELL" name="com.w2a.architectbatch1.testCases.TestCase1" tests="1" failures="0" timestamp="04 Nov 2017 08:05:12 GMT" time="16.879" errors="1">
  <testcase name="tc_02" time="16.879" classname="com.w2a.architectbatch1.testCases.TestCase1">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:238)
	at com.w2a.architectbatch1.Pages.BasePage.waitForPageToLoad(BasePage.java:50)
	at com.w2a.architectbatch1.Pages.BasePage.openPage(BasePage.java:39)
	at com.w2a.architectbatch1.Pages.FacebookHomePage.open(FacebookHomePage.java:10)
	at com.w2a.architectbatch1.Pages.FacebookLandingPage.doLoginWithValidCredentials(FacebookLandingPage.java:36)
	at com.w2a.architectbatch1.testCases.TestCase1.tc_02(TestCase1.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- tc_02 -->
</testsuite> <!-- com.w2a.architectbatch1.testCases.TestCase1 -->
